### DISCUSSION TITLE
Axum and tracing to a file

### DISCUSSION THREAD

**Discussion by kellytk** [tokio-rs/axum#752] [Q&A] [answered]
I have an Axum-based project successfully tracing to a file with one exception, request tracing.

I've followed the [tracing-aka-logging](https://github.com/tokio-rs/axum/tree/main/examples/tracing-aka-logging) example to make https://github.com/kellytk/axum-trace.  The code runs as expected tracing to the terminal, however if I comment out [this line](https://github.com/kellytk/axum-trace/blob/main/src/main.rs#L6) and uncomment the following line, request handling traces do not appear in the file.

How please should it be correctly configured?

**Comment by jplatte** [comment, marked as answer]
This question seems more appropriate for https://github.com/tokio-rs/tracing/discussions, no?

**Reply by kellytk** [reply]
[Done](https://github.com/tokio-rs/tracing/discussions/1916)
